mlrun/api/db/filedb/db.py

```diff
@@ -134,16 +134,29 @@ def list_functions(self, session, name=None, project="", tag="", labels=None):
     def store_schedule(self, session, data):
         return self._transform_run_db_error(self.db.store_schedule, data)
 
-    def list_projects(self, session, owner=None):
-        return self._transform_run_db_error(self.db.list_projects)
+    def list_projects(
+        self, session, owner: str = None, full: bool = True
+    ) -> schemas.ProjectsOutput:
+        return self._transform_run_db_error(self.db.list_projects, owner, full)
 
-    def add_project(self, session, project: dict):
+    def store_project(self, session, name: str, project: schemas.Project):
         raise NotImplementedError()
 
-    def update_project(self, session, name, data: dict):
+    def patch_project(
+        self,
+        session,
+        name: str,
+        project: schemas.ProjectPatch,
+        patch_mode: schemas.PatchMode = schemas.PatchMode.replace,
+    ):
+        raise NotImplementedError()
+
+    def create_project(self, session, project: schemas.Project):
         raise NotImplementedError()
 
-    def get_project(self, session, name=None, project_id=None):
+    def get_project(
+        self, session, name: str = None, project_id: int = None
+    ) -> schemas.Project:
         raise NotImplementedError()
```

Comment:
It will be removed soon
It is somewhat viable in the scenario of just running the mlrun API without UI with jupyter (the jupyter image) to play with MLRun (only local runtime)